src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ProductListHeader from './ProductListHeader/ProductListHeader';
+import ProducListContent from './ProducListContent/ProducListContent';
 import './ProductList.scss';
+
 function ProductList() {
-  return <div></div>;
+  const [curCategoty, setCurCategoty] = useState(0);
+  const [products, setProducts] = useState([]);
+
+  const PRODUCT_MENU = {
+    menuName: '채소',
+    categories: ['전체보기', '브로컬리', '쌈채소', '간편채소'],
+  };
+
+  useEffect(() => {
+    fetch(`data/productListData${curCategoty}.json`)
+      .then(res => {
+        return res.json();
+      })
+      .then(res => {
+        setProducts(res);
+      });
```

Comment:
```suggestion
      .then(res => res.json())
      .then(res => setProducts(res));
```
arrow function 문법에서 생략 가능한 부분들 잘 확인하면 훨씬 간결하게 작성할 수 있습니다!

Reply:
수정했습니다. 감사합니다!